libnopegl/src/nopegl.h.in

```diff
@@ -118,11 +132,19 @@ struct ngl_scene {
 NGL_API struct ngl_scene *ngl_scene_create(void);
 
 /**
- * Initialize the scene graph passing down the graph directly.
+ * Initialize the scene from parameters.
+ *
+ * The reference counter of the root node will be incremented for the scene
+ * lifetime, or until another scene replaces it.
```

Comment:
Same here, @param + @return.

Reply:
added a `@return` but no `@param`, it's obvious what `params` is, it would just add clutter.

Reply:
Yes it is obvious but I would prefer to make it consistent with the majority of the doc in this file.

Something like that would be enough:
`* Initialize a scene` + `@params` pointer to a scene to initialize`

Reply:
done